From leoli@freescale.com  Thu Mar  6 10:21:08 2008
From: Li Yang <leoli@freescale.com>
Date: Thu,  6 Mar 2008 18:40:07 +0800
Subject: USB: fsl_usb2_udc: fix broken Kconfig
To: greg@kroah.com
Cc: david-b@pacbell.net, linux-usb@vger.kernel.org, Li Yang <leoli@freescale.com>
Message-ID: <1204800007-11568-1-git-send-email-leoli@freescale.com>


The patch fixes broken Kconfig caused by the name change of MPC834x option.
It also makes fsl_usb2_udc selectable on new platforms like MPC837x.

Signed-off-by: Li Yang <leoli@freescale.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/gadget/Kconfig
+++ b/drivers/usb/gadget/Kconfig
@@ -131,7 +131,7 @@ config USB_ATMEL_USBA
 
 config USB_GADGET_FSL_USB2
 	boolean "Freescale Highspeed USB DR Peripheral Controller"
-	depends on MPC834x || PPC_MPC831x
+	depends on FSL_SOC
 	select USB_GADGET_DUALSPEED
 	help
 	   Some of Freescale PowerPC processors have a High Speed
